GVFS/GVFS/CommandLine/HealthVerb.cs

```diff
@@ -67,23 +68,42 @@ protected override void Execute(GVFSEnlistment enlistment)
 
         private void PrintOutput(EnlistmentHealthData enlistmentHealthData)
         {
-            string trackedFilesCountFormatted = enlistmentHealthData.GitTrackedItemsCount.ToString("N0");
-            string placeholderCountFormatted = enlistmentHealthData.PlaceholderCount.ToString("N0");
-            string modifiedPathsCountFormatted = enlistmentHealthData.ModifiedPathsCount.ToString("N0");
-
-            // Calculate spacing for the numbers of total files
-            int longest = Math.Max(trackedFilesCountFormatted.Length, placeholderCountFormatted.Length);
-            longest = Math.Max(longest, modifiedPathsCountFormatted.Length);
-
-            // Sort the dictionary to find the most hydrated directories by health score
+            // Get the appropriate number of sub directories to display
             List<KeyValuePair<string, EnlistmentHealthCalculator.SubDirectoryInfo>> topLevelDirectoriesByHydration = enlistmentHealthData.DirectoryHydrationLevels.Take(this.DirectoryDisplayCount).ToList();
 
-            this.Output.WriteLine("\nHealth of directory: " + enlistmentHealthData.TargetDirectory);
-            this.Output.WriteLine("Total files in HEAD commit:           " + trackedFilesCountFormatted.PadLeft(longest) + " | 100%");
-            this.Output.WriteLine("Files managed by VFS for Git (fast):  " + placeholderCountFormatted.PadLeft(longest) + " | " + this.FormatPercent(enlistmentHealthData.PlaceholderPercentage));
-            this.Output.WriteLine("Files managed by Git:                 " + modifiedPathsCountFormatted.PadLeft(longest) + " | " + this.FormatPercent(enlistmentHealthData.ModifiedPathsPercentage));
-
-            this.Output.WriteLine("\nTotal hydration percentage:           " + this.FormatPercent(enlistmentHealthData.PlaceholderPercentage + enlistmentHealthData.ModifiedPathsPercentage).PadLeft(longest + 7));
+            string targetDirectoryLabel = "Health of directory: " + enlistmentHealthData.TargetDirectory;
+            string totalFilesLabel = "Total files in HEAD commit:";
+            string placeholderLabel = "Files managed by VFS for Git (faster):";
+            string modifiedPathsLabel = "Files managed by Git:";
+
+            string fileLabel = "File";
+            string directoryLabel = "Dir";
+            string totalLabel = "Sum";
+
+            int labelWidth = Math.Max(Math.Max(targetDirectoryLabel.Length, totalFilesLabel.Length), Math.Max(placeholderLabel.Length, modifiedPathsLabel.Length)) + 1;
+            int fileWidth = Math.Max(enlistmentHealthData.GitTrackedFileCount.ToString("N0").Length, fileLabel.Length);
+            int directoryWidth = Math.Max(enlistmentHealthData.GitTrackedFolderCount.ToString("N0").Length, directoryLabel.Length);
+            int totalWidth = Math.Max(enlistmentHealthData.GitTrackedItemsCount.ToString("N0").Length, totalLabel.Length);
+
+            string formattedTrackedFiles = enlistmentHealthData.GitTrackedFileCount.ToString("N0").PadLeft(fileWidth);

```

Comment:
Maybe put this behind a method?

`string FormatForColumn(int count, int width) { ... }`